### PR TITLE
i18n(fr): create 404 french version

### PR DESCRIPTION
**Pull request by cbontems**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description
 
- What does this PR change? Give us a brief description.
Created missing french version of 404.md

### CODE REVIEW

File: docs/src/content/docs/fr/404.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Page Introuvable
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: <strong>Houston, nous avons un problème.</strong> Nous n’avons pas pu trouver cette page.<br>Vérifiez l’URL ou essayez la barre de recherche.
+  actions:
+    - text: Retourner à l’accueil
+      icon: right-arrow
+      link: /
```
Comment (delucis) [inline docs/src/content/docs/fr/404.md:11]: Thanks for this contribution @cbontems!

Looks great. Here we can link to the French homepage though:
```suggestion
      link: /fr/
```

**Comment by cbontems** [comment]
Sure! I missed the link back... Thanks @delucis for the suggestion.

**Review by delucis — APPROVED** [review]
Thanks again @cbontems!

**Comment by cbontems** [comment]
> Thanks again @cbontems!

My pleasure!

**Pull request merged**